worker.py

```diff
@@ -25,5 +27,9 @@
 broker.add_middleware(InspectorMiddleware(os.environ["RPC_URL"]))
 dramatiq.set_broker(broker)
 
-dramatiq.actor(inspect_many_blocks_task, queue_name=HIGH_PRIORITY_QUEUE)
-dramatiq.actor(export_block_task, queue_name=LOW_PRIORITY_QUEUE)
+dramatiq.actor(
+    inspect_many_blocks_task, queue_name=LOW_PRIORITY_QUEUE, priority=LOW_PRIORITY
+)
+dramatiq.actor(
+    export_block_task, queue_name=HIGH_PRIORITY_QUEUE, priority=HIGH_PRIORITY
```

Comment:
hmmm i wonder how this will play if we want low priority for backfill high priority for live - might need another task after all to separate the two